src/deepsparse/loggers/metric_functions/computer_vision/built_ins.py

```diff
@@ -19,6 +19,8 @@
 
 import numpy
 
+from deepsparse.loggers.metric_functions import register
```

Comment:
nice - like how easy it is to construct a set of functions for a task. need to now think about how to manage when registry happens - especially due to dependencies (ie will this file always be imported with the `loggers` submodule so registration fires?

Reply:
Good point. In the incoming commit, I will be probably postponing the actual registration until the logic of logger building encounters a data logging config target that will be associated with loading the predefined configuration. Stay tuned.